### ISSUE TITLE
Don't autocomplete to <> when you type <

### ISSUE THREAD

**Issue by spencerchubb** [rust-lang/vscode-rust#902]
I'm having a minor annoyance. If I'm trying to use the less than operator it autocompletes to make a pair of angle brackets, and then I have to delete the extra closing bracket that was created.

What I want: `<`
What I get: `<>`

Is there a configuration I can use to turn this off for me personally or can this feature be changed for the VS Code extension?

**Comment by Drarig29** [comment]
I didn't find such a setting either.

**Comment by bmd7297** [comment]
Dear god someone please fix this it's so annoying. Like, yes, it isn't the end of the world, but it is one of the many issues I have with this extension that keeps me from fully enjoying the Rust language.

**Comment by lnicola** [comment]
@bmd7297 If you haven't already, maybe give https://marketplace.visualstudio.com/items?itemName=matklad.rust-analyzer a try.

**Comment by bmd7297** [comment]
> @bmd7297 If you haven't already, maybe give https://marketplace.visualstudio.com/items?itemName=matklad.rust-analyzer a try.

Yeah I'm just now trying it for a myriad of reasons. Thanks for the recommendation.
